lib/vmfloaty.rb

```diff
@@ -256,10 +272,13 @@ def run # rubocop:disable Metrics/AbcSize
         end
 
         unless successes.empty?
-          FloatyLogger.info unless failures.empty?
```

Comment:
This might need to stay in for formatting reasons? For a clean line break. If the change to group up the message into a single string happens like I mentioned below, you can just prepend the beginning of the grouped message with a `\n` instead.